pom.xml

```diff
@@ -451,6 +452,18 @@
       <artifactId>junit</artifactId>
       <version>${junit.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.powermock</groupId>
```

Comment:
why was this dependency added?

Reply:
In release/0.15 we would get an error while building stating that this dependency was missing. We think that it might be missing from this PR: https://github.com/data-integrations/google-cloud/pull/627

Should we create a new PR for this build fix?

Reply:
I see, thanks.